src/AnalyzeRegistry.jl

```diff
@@ -292,18 +301,21 @@ these can't be inferred from the source code. If `name` or `uuid` are `nothing`,
 directories `Project.toml` is parsed to infer the package's name and UUID.
 
 ## Example
-```julia
-julia> analyze(pkgdir(AnalyzeRegistry))
-Package AnalyzeRegistry:
-  * repo: 
-  * uuid: e713c705-17e4-4cec-abe0-95bf5bf3e10c
+
+```jldoctest
```

Comment:
I'm not entirely sure it's a good idea to make these doctests: the output will change over time

Reply:
yeah, fair; my thought was then that would cause us to keep them up-to-date (with `doctest(AnalyzeRegistry; fix=true)`) but I can see how that's annoying too, so I got rid of them. (All the doctests were things that can go out of date easily)

Reply:
The annoying thing is that they are out of our control, tests would fail just because `DataFrames` published a new version, not because there is something broken in this package.